event/listener.php

```diff
@@ -78,7 +78,6 @@ static public function getSubscribedEvents()
 
 			// Message parsing
 			'core.modify_text_for_display_before'			=> 'replace_mentions_for_display',
-			'core.modify_text_for_display_before'			=> 'replace_mentions_for_display',
```

Comment:
Uh what. I am really too tired today to work on extensions ._.
I think I meant that, yes. There are two events. I'll test that out, but you are right I would say.